README.md

```diff
@@ -22,60 +22,175 @@
+To complete this guide, you will need a Kubernetes cluster (with a suggested minimum of 6 vCPUs and 8 GB RAM). You can run the cluster locally using Kind or Minikube.
 
-## Documentation
+> The quick start will help you set up a functioning Kafka cluster on Kubernetes. However, it does not include guidance on the installation of Prometheus and cert-manager, which are necessary for some of the more advanced functionality.
+
+#### Install ZooKeeper
+
+The version of Kafka that is installed by the operator requires Apache ZooKeeper. You'll need to deploy a ZooKeeper cluster if you don’t already have one.
+
+1. Install ZooKeeper using [Pravega’s Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+
+```
+helm repo add pravega https://charts.pravega.io
+helm repo update
+helm install zookeeper-operator --namespace=zookeeper --create-namespace pravega/zookeeper-operator
+```
+
+2. Create a ZooKeeper cluster.
+
+```
+kubectl create --namespace zookeeper -f - <<EOF
+apiVersion: zookeeper.pravega.io/v1beta1
+kind: ZookeeperCluster
+metadata:
+    name: zookeeper
+    namespace: zookeeper
+spec:
+    replicas: 1
+EOF
+```
+
+3. Verify that ZooKeeper has been deployed.
+
+```
+> kubectl get pods -n zookeeper
+
+NAME                                  READY   STATUS    RESTARTS   AGE
+zookeeper-0                           1/1     Running   0          27m
+zookeeper-operator-54444dbd9d-2tccj   1/1     Running   0          28m
+```
+
+### Install Koperator
+
+You can deploy Koperator using a Helm chart. Complete the following steps.
 
-The documentation of the Koperator project is available at the [Banzai Cloud Documentation Page](https://banzaicloud.com/docs/supertubes/kafka-operator/).
+1. Install the Koperator `CustomResourceDefinition` resources (adjust the version number to the Koperator release you want to install). This is performed in a separate step to allow you to uninstall and reinstall Koperator without deleting your already installed custom resources.
 
-## Support
+```
+kubectl create --validate=false -f https://github.com/banzaicloud/koperator/releases/download/v0.24.0/kafka-operator.crds.yaml
+```
 
-### Community support
+2. Add the following repository to Helm.
 
-If you encounter problems while using Koperator that the documentation does not address, [open an issue](https://github.com/banzaicloud/koperator/issues) or talk to us on the Banzai Cloud Slack channel [#kafka-operator](https://banzaicloud.com/invite-slack/).
+```
+helm repo add banzaicloud-stable https://kubernetes-charts.banzaicloud.com/
+helm repo update
+```
 
-### Commercial support
+3. Install Koperator into the `kafka` namespace:
 
-If you are using Koperator in a production environment and [require commercial support, contact Banzai Cloud](https://banzaicloud.com/contact/), the company backing the development of Koperator.
+```
+helm install kafka-operator --namespace=kafka --create-namespace banzaicloud-stable/kafka-operator
+```
```

Comment:
**Recommended**: We **SHOULD** consider using a more concise command list, there is no need for local tracking - and manual updating - of the remote Helm chart repository's index.

`helm install kafka-operator --repo https://kubernetes-charts.banzaicloud.com/ --namespace=kafka --create-namespace banzaicloud-stable/kafka-operator`

(can't suggest directly due to limitations)